### ISSUE TITLE
[BITV] 9.1.3.1b/5.2 - The lists of links that have been implemented as icons (see screenshot) should be properly structured with corresponding HTML list elements or ARIA roles. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41831]
https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-1d5eadb58d-v5-n2

![image](https://github.com/nextcloud/server/assets/6078378/34b498f5-7a11-4c8c-826e-b8e384c464db)

**Comment by ShGKme** [comment]
> Who is aria? please describe the behavior expected

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA
